launch/tier4_localization_launch/launch/util/util.launch.py

```diff
@@ -87,24 +87,32 @@ def add_launch_arg(name: str, default_value=None, description=None):
         arg = DeclareLaunchArgument(name, default_value=default_value, description=description)
         launch_arguments.append(arg)
 
+    add_launch_arg(
+        "tier4_localization_launch_param_path",
+        [FindPackageShare("tier4_localization_launch"), "/config"],
```

Comment:
I put the default parameter None. What do you think?
https://github.com/tier4/autoware.universe/blob/5a13fa38f46e87b5423264bafff3851ad229c69b/launch/tier4_planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py#L487

Reply:
I think it would be better to provide the default parameter in this particular case (at least for now), since the original `util.launch.py` also provides the default parameter for `crop_box_filter_measurement_range_param_path`, `voxel_grid_downsample_filter_param_path`, etc. 
Removing the default parameter would change the functionality of the launch file, so IMO it's better to leave it as it is for now.

Reply:
Got it, thanks